app/controllers/api/v1/base_controller.rb

```diff
@@ -240,8 +259,11 @@ def key_transform_header
       private
 
       def authenticate_user_token!
-        authenticate_with_http_token do |token, _options|
-          return User.includes(:language).find_by_auth_token(token)
+        authenticate_with_http_token do |auth_token, _options|
+          token = Token.includes(:user).find_by(token: auth_token)
```

Comment:
☝️ scratch that the block doesn't yield if `auth_token` is either `nil` or `blank?`.